packages/booter-lb3app/src/lb3app.booter.ts

```diff
@@ -62,6 +63,22 @@ export class Lb3AppBooter implements Booter {
       this.mountRoutesOnly(lb3App, spec);
     }
 
+    const dataSources = lb3App.dataSources;
+    if (dataSources) {
+      const processedDs: string[] = [];
+      Object.keys(dataSources).forEach(key => {
+        key = key.toLowerCase();
+        if (!processedDs.includes(key)) {
+          const ds = dataSources[key];
+          processedDs.push(ds.name);
+          this.app
+            .bind(`datasources.lb3-${ds.name}`)
```

Comment:
I would like to discuss the binding naming convention with @raymondfeng and possibly with the wider community @strongloop/loopback-next .

Aspects to consider:
- There should be very low probability of a naming conflict between a key generated for a LB3 datasource and a key created by LB4 API `app.dataSource`.
- The key should make it obvious what's the datasource name and that this datasource is coming from a LB3 app.

I think the current proposal `datasources.lb3-${ds.name}` is reasonable 👍 

Few other options to consider:
- `lb3-datasources.${ds.name}` (e.g. `lb3-datasources.db`)
- `datasources.\$lb3\$${ds.name}` (e.g. `datasources.$lb3$db`)

Reply:
I am good with either 
`datasources.lb3-${ds.name}` (pro: the prefix is consistent with other datasources)
or 
`lb3-datasources.${ds.name}` (pro: we can apply the same naming convention for other lb3 artifacts like models)